config.json

```diff
@@ -27,5 +27,14 @@
   },
   "concepts": [],
   "key_features": [],
-  "tags": []
+  "tags": [
```

Comment:
Thank you for filling this list! :+1: I have some doubts about 2 items though:
* according to the [official documentation](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html), GDScript is "gradually typed", not "dynamically typed"; I am not sure if it makes any difference for us (I mean, if "gradual" is not an allowed value for Configlet, I guess we might as well use "dynamic")
* I don't think that GDScript is "interpreted", even though it looks a lot like Python ... I believe the code is compiled before executing it, and there is also no such thing as a REPL in Godot; perhaps we can ask someone who spent more time maintaining the code about the true nature of GDScript ;)

Reply:
No gradual typing tag. Here are all the options available under typing.

Typing
-     `typing/static`: the language uses static typing
-     `typing/dynamic`: the language uses dynamic typing
-     `typing/strong`: the language uses strong typing
-     `typing/weak`: the language uses weak typing

For what it's worth, https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_advanced.html indicates GDScript is dynamically typed and that no compilation is needed for testing. The compilation seems to occur when you export the code out of Godot, but given the Exercism building docs say "a track should choose their tags based on the general usage of their language", I think it's appropriate to consider GDScript an interpreted language since that's how it presents within Godot.

Reply:
Agreed, thanks for digging into this! :)

Reply:
> I mean, if "gradual" is not an allowed value for Configlet, I guess we might as well use "dynamic"

There are basically two ways to look at this (gradual typing):

- Use both the `static` _and_ `dynamic` tags
- We introduce a new `gradual` tag

@ee7 Thoughts?

Reply:
@ee7 I've changed the typing to `gradual` in the "Hello, World!" PR (#3 ) for now. If we decide not to add this new typing to allowed Configlet values, let me know, I will add both `static` and `dynamic` (as suggested by @ErikSchierboom ).